### PR TITLE
Sdks 3615 handle 302

### PR DESCRIPTION
**Pull request by vahancouver**
[SDKS-3615](https://pingidentity.atlassian.net/browse/SDKS-3615)
[iOS] Throws Unknown error instead of APIException when server return 302 with empty body

I also fixed the 2 following issues in that branch:
- The subprojects/libraries wouldn’t compile automatically
- Token and user info views would hang the app sometimes.

### CODE REVIEW

File: SampleApps/PingExample/PingExample/AccessToken.swift [hunk 1]
```diff
@@ -16,27 +16,26 @@ struct AccessTokenView: View {
     
     var body: some View {
         VStack {
-            
-            TextEditor(text: $accessToken.accessToken)
-                            .foregroundStyle(.secondary)
-                            .padding(.horizontal)
-                            .navigationTitle("AccessToken")
+            ScrollView {
```
Comment (rodrigoareis) [inline SampleApps/PingExample/PingExample/AccessToken.swift:19]: I believe we decided to rename the folder of sample apps to something like `testApps` or `e2e`, right? Should we do it now?

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

**Review by jeyanthanperiyasamy — APPROVED** [review]

File: SampleApps/PingExample/PingExample/AccessToken.swift (hunk 1 above)
Comment (vahancouver) [reply SampleApps/PingExample/PingExample/AccessToken.swift:19]: @rodrigoareis we have decided for the FR repo and there is a ticket in this sprint for that (to name it e2e). I'm not aware of a similar decision for the Pings SDKs.

**Review by spetrov — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**